### PR TITLE
widelands 1.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
Patch can be removed

File: Formula/widelands.rb [hunk 1]
```diff
@@ -35,23 +36,23 @@ class Widelands < Formula
...
     ENV.cxx11
-    system "cmake", "-S", ".", "-B", "build",
-                    # Without the following option, Cmake intend to use the library of MONO framework.
-                    "-DPNG_PNG_INCLUDE_DIR:PATH=#{Formula["libpng"].opt_include}",
-                    "-DWL_INSTALL_DATADIR=#{pkgshare}/data",
-                    # older versions of macOS may not have `python3`
-                    "-DPYTHON_EXECUTABLE=#{which("python3") || which("python")}",
-                    *std_cmake_args
+
+    cmake_args = std_cmake_args + %w[
```
Comment (carlocab) [inline Formula/widelands.rb:44]:
```suggestion
    cmake_args = %W[
```

File: Formula/widelands.rb [hunk 2]
```diff
@@ -35,23 +36,23 @@ class Widelands < Formula
...
+      -DOPTION_BUILD_TESTS=OFF
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DWL_INSTALL_BASEDIR=#{pkgshare}
+      -DWL_INSTALL_BINDIR=#{bin}
+      -DWL_INSTALL_DATADIR=#{pkgshare}/data
+      -DPNG_PNG_INCLUDE_DIR:PATH=#{Formula["libpng"].opt_include}
+      -DPYTHON_EXECUTABLE=#{which("python3") || which("python")}
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *cmake_args
```
Comment (carlocab) [inline Formula/widelands.rb:55]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *cmake_args, *std_cmake_args
```
Always pass `std_cmake_args` last to make sure you don't override it

File: Formula/widelands.rb [hunk 3]
```diff
@@ -20,7 +19,9 @@ class Widelands < Formula
     sha256 x86_64_linux:   "1032792700cccdbaa04827dbc6849c309fa75d649f3253bd0c7568a1fd5d6770"
   end
 
+  depends_on "asio" => :build
   depends_on "cmake" => :build
+
   depends_on "boost"
```
Comment (carlocab) [inline Formula/widelands.rb:25]: Do we still need Boost?

File: Formula/widelands.rb [hunk 4]
```diff
@@ -20,7 +19,9 @@ class Widelands < Formula
...
   depends_on "doxygen"
```
Comment (carlocab) [inline Formula/widelands.rb:26]: Runtime dep on Doxygen is odd.

Comment (chenrui333) [reply Formula/widelands.rb:26]: yeah, indeed.

File: Formula/widelands.rb (hunk 3 above)
Comment (chenrui333) [reply Formula/widelands.rb:25]: per [this issue](https://github.com/widelands/widelands/issues/5115), no, let me update the list and make some run

**Comment by klaus-halfmann** [comment]
@carlocab  / @chenrui333 any idea why this got stale? 
See also https://github.com/orgs/Homebrew/discussions/3924

**Comment by chenrui333** [comment]
> @carlocab / @chenrui333 any idea why this got stale? See also https://github.com/orgs/Homebrew/discussions/3924

running into build issues, feel free to file a new PR continuing this work.

**Comment by klaus-halfmann** [comment]
After fixing https://github.com/widelands/widelands/issues/5655 I will try again

**Pull request closed without merging**